extensions/wikia/AdMixExperiment/scripts/AdMixExperiment.js

```diff
@@ -25,8 +27,14 @@ require(['ext.wikia.adEngine.adContext', 'wikia.abTest', 'wikia.throttle'], func
 		var visibleElementBeforeWrapperHeight;
 
 		// ad mix flags
-		var topRightAdFixed = !!(abTest.getGroup('AD_MIX') && abTest.getGroup('AD_MIX') !== 'AD_MIX_2');
-		var recircEnabled = !!(abTest.getGroup('AD_MIX') && abTest.getGroup('AD_MIX') !== 'CONTROL');
+		// AD_MIX_1: reloadFloatingMedrec === true && recircEnabled === true
+		// AD_MIX_2: topRightAdFixed === true && recircEnabled === true
+		// AD_MIX_3: reloadRecirc === true && recircEnabled === true
+		// CONTROL: all === false
+		var reloadFloatingMedrec = abTest.getGroup('AD_MIX') === 'AD_MIX_1';
+		var topRightAdFixed = abTest.getGroup('AD_MIX') === 'AD_MIX_2';
+		var reloadRecirc = abTest.getGroup('AD_MIX') === 'AD_MIX_3';
+		var recircEnabled = abTest.getGroup('AD_MIX') !== 'CONTROL';
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "recircEnabled" as this name is already used in declaration at line 11. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2814)